src/shared/lib/helpers/truncateMiddle.ts

```diff
@@ -1,4 +1,8 @@
 export const truncateMiddle = (str: string, sliceLength = 5, separator = '...') => {
```

Comment:
```suggestion
export const truncateMiddle = (str: string | undefined, sliceLength = 5, separator = '...') => {
```
или что-то типа того, раз функция теперь обрабатывает пустое значение